Dockerfile

```diff
@@ -11,4 +11,10 @@ RUN make install
 FROM alpine:3.15.4@sha256:4edbd2beb5f78b1014028f4fbb99f3237d9561100b6881aabbf5acce2c4f9454
 COPY --from=build /go/bin/metacontroller /usr/bin/metacontroller
 RUN apk update && apk add --no-cache ca-certificates
+
+# Run container as nonroot, use the same uid and naming convention as distroless images
+# See https://github.com/GoogleContainerTools/distroless/blob/0d757ece34cdc83a2148cea6c697e262c333cb84/base/base.bzl#L8
+RUN addgroup -g 65532 -S nonroot && adduser -D -u 65532 -g nonroot -S nonroot -G nonroot
```

Comment:
if the change is here, it should be also in https://github.com/metacontroller/metacontroller/blob/master/Dockerfile.distroless

Reply:
and https://github.com/metacontroller/metacontroller/blob/master/Dockerfile.distroless-debug

Reply:
also, if you are changing dockerfile, regular manifests in https://github.com/metacontroller/metacontroller/blob/master/manifests/production/metacontroller.yaml#L21 also requires similar change as helm chart

Reply:
Hi @grzesuav , for the distroless images, they actually have the nonroot user created by default in the base image `gcr.io/distroless/base-debian10`. The Dockerfile are already setting `USER nonroot:nonroot` so I don't need to create the user/group manually in this case.

For https://github.com/metacontroller/metacontroller/blob/master/manifests/production/metacontroller.yaml#L21 no security context is provided and `runAsUser` is omitted. So, it will default to the `USER` defined in the Dockerfile. So, I shouldn't need to change that file as well. I did add a helm test explicitly setting the value https://github.com/metacontroller/metacontroller/pull/534/files#diff-b9b441d47a7c034fadc81d16dc1f0ab6dabcf005b4e4d60fcca4df17162f6a7cR32. 

Please let me know if you have any questions/concerns.